src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/steps/CoverageBuildAction.java

```diff
@@ -84,17 +78,10 @@ public final class CoverageBuildAction extends BuildAction<Node> implements Stap
     /** The coverage delta of the associated change request with respect to the reference build. */
     private NavigableMap<Metric, Fraction> modifiedLinesCoverageDifference;
```

Comment:
We can remove `modifiedLinesCoverageDifference` field and its jelly template parts here. The jelly template file is already deleted in  [cicd\_474 && cicd\_490 by dtang-parasoft · Pull Request #56 · jenkinsci/parasoft-findings-plugin](https://github.com/jenkinsci/parasoft-findings-plugin/pull/56/files#diff-0efc50763b429e72f20face3f07e06894686a330c5ec889f093018c1430c746d)